packages/json-schema/src/validate.ts

```diff
@@ -630,7 +645,6 @@ export function validate(
             },
             ...result.errors
           );
-          break;
```

Comment:
Would it be possible/better to put the `if (stop) break;` here to reduce the number of times that line of code executes?

Reply:
Yep, agreed. Thanks for spotting

Reply:
Moved, thanks!